server/cmd/mmctl/commands/auth_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/printer"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
+	"os/user"
+	"path/filepath"
+)
```

Comment:
CI is unhappy with the import order: 
```
/home/runner/work/mattermost/mattermost/server/bin/golangci-lint run ./...
cmd/mmctl/commands/auth_test.go:7: File is not `goimports`-ed (goimports)
	"github.com/mattermost/mattermost/server/v8/cmd/mmctl/printer"
	"github.com/spf13/cobra"
	"github.com/spf13/viper"
make: *** [Makefile:289: golangci-lint] Error 1
```

Please use the same order as in https://github.com/annaos/mattermost/blob/ee6be981c952791fe2de556e9964d47e2b77e796/server/cmd/mmctl/commands/channel_users_e2e_test.go#L6-L16.